visualisations/fan_chart/webviz_fan_chart/__init__.py

```diff
@@ -1,33 +1,34 @@
 from webviz_plotly import Plotly
 import pandas as pd
 import matplotlib.cm as cm
+import pdb
 
 color_scheme = cm.get_cmap('Set1')
 
 
 def color_spread(n):
     """Color generator function
 
-    :param data:
-        n = number of total colors needed
-        Returns list of lists of three strings representing rgb
+    :param n:
+        n = object containing all separate line names
+        Returns dictionary with format {'line-name': ['r','g','b']}
     """
-    colorlist = []
-    for i in range(n):
-        single_color = color_scheme(float(i) / n)
+    colorlist = {}
+    for i, name in enumerate(n):
+        single_color = color_scheme(float(i) / len(n))
         formatted_color = []
         for y in single_color[:-1]:
             formatted_color.append(str(y))
-        colorlist.append(formatted_color)
+        colorlist[name] = formatted_color
     return colorlist
 
 
 def format_color(color, opacity):
     """Color formatting function
 
-    :param data:
-        color list with three strings to represent an rgb color,
-        opacity between 0 and 1.
+    :param
```

Comment:
This is not valid rst as far as I know. Currently this only passes our build since FanChart is not added to the documentation. Could you add FanChart to the list of visualisations in `docs/`?

Reply:
It was added when FanChart first was created without observations.
Got the suggested format from PyCharm, but I can write it differently if preferred.